modal_proto/api.proto

```diff
@@ -1712,15 +1713,16 @@ message RuntimeOutputMessage {
 }
 
 message RuntimeOutputBatch {
-  string entry_id = 1;
+  string entry_id = 1 [deprecated=true];
```

Comment:
We don't need to deprecate these fields since there's no one actively using them! You can just remove them :)

Reply:
I saw @luiscape's comment above